apps/atqr/web/src/services/interfaces.ts

```diff
@@ -1,31 +1,51 @@
-export enum SupervisorEnum {
-    'notInvited',
-    'invited',
-    'accepted',
-    'askedIfTheGoalIsAccomplished',
-    'repliedIfTheGoalWasAccomplished',
+  export enum ChallengeStatus {
```

Comment:
These enums are in here AND in enums.ts. Please, delete these duplicates in interfaces.ts.

Reply:
Yep!

Reply:
Done!